scripts/collect-issues.js

```diff
@@ -0,0 +1,33 @@
+import ghGot from 'gh-got'
+import { buildRFC822Date, composeFeedItem, getFeedContent, overwriteFeedContent, getConfig } from '../utils/index.js'
+
+const { lastCheckTimestamp, breakDelimiter, issuesInScope, commentsPaginationLimit } = getConfig()
+
+// Collect all the comments for the issues in scope
+const comments = await Promise.all(issuesInScope.map(async issue => {
+  const issueComments = await ghGot(
+              `repos/${issue}/comments`,
+              {
+                token: process.env.GITHUB_TOKEN,
+                pagination: { countLimit: commentsPaginationLimit }
+              }
+  ).json()
+
+  // Select only comments that are newer than the last check and add issue context
+  return issueComments
+    .filter(comment => new Date(comment.updated_at).getTime() > lastCheckTimestamp)
+    .map(comment => ({ ...comment, issue }))
+}))
+
+const relevantComments = comments.flat().map(comment => composeFeedItem({
+  title: 'TODO: REVIEW THIS COMMENT',
```

Comment:
Is this because we don't have a title for a comment? We might just make them something like uvwasi team update on DD MM YY HH MM SS.

Reply:
Yes! I like the approach, I will patch it

Reply:
I made some changes in [118d6cb](https://github.com/UlisesGascon/poc-nodejs-news-feeder/pull/5/commits/118d6cb6bad26ba6dbb19ff3d805e71ea2c05895) and now the titles are quite more friendly:

```xml
<item>
  <title>News Feeder team update on 2023-06-13 21:16:09</title>
</item>
<item>
  <title>Uvwasi team update on 2023-06-07 18:05:55</title>
</item>
<item>
  <title>Node-API and node-addon-api teams update on 2023-06-02 15:09:28</title>
</item>
```